Sources/InternalCollectionsUtilities/Descriptions.swift.gyb

```diff
@@ -45,7 +45,9 @@ ${modifier} func _arrayDescription<C: Collection>(
     } else {
       result += ", "
     }
+#if !$Embedded
     debugPrint(item, terminator: "", to: &result)
+#endif
```

Comment:
Hm, this replaces `[1, 2, 3]` with `[, , ]` in `description` properties, and that's not very nice.

In this case, the stdlib is currently [substituting a generic message for each item](https://github.com/swiftlang/swift/blob/main/stdlib/public/core/ArrayShared.swift#L124-L128), like this:
```suggestion
#else
    result += "(unprintable)"
#endif
```

This results in `[(unprintable), (unprintable), (unprintable)]`. To be fair, that isn't that useful, either. I think it would be preferable to just replace the entire body with a short summary like `"[\(count) values]"`. We can also choose to entirely omit the `CustomStringConvertible` conformances under `$Embedded`.

(Ditto for `_dictionaryDescription` and all other collection printing code below.)

Reply:
Sounds good.

Reply:
This turns out to be a good idea because the `LosslessStringConvertible` conformance also relied on description.